lib/master.js

```diff
@@ -15,8 +15,19 @@ var util = require('util');
 // Master is a singleton, as is the cluster master
 var master = new EventEmitter();
 
+// Workers remain in cluster.workers after they have have exited.
+function liveWorkerIds() {
+  var workerIds = Object.keys(cluster.workers).filter(function(id) {
+    var worker = cluster.workers[id];
+    var exited = worker.process.signalCode != null ||
```

Comment:
I think this would be easier to follow:
```js
var isAlive = worker.process.signalCode == null && worker.process.exitCode == null;
return isAlive;
```